<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 150, 150);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="settingWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(120, 120, 120);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1,1,1,1,1,10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="settingLabel_5">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>设置setting</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="modelLabel_5">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Berlin Sans FB&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>模型选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modelCombo_5">
           <property name="styleSheet">
            <string notr="true">QComboBox{
	color: rgb(255, 255, 255);
	font: 9pt &quot;Agency FB&quot;;
}
QComboBox QAbstractItemView {
    color: white; 
}</string>
           </property>
           <item>
            <property name="text">
             <string>yolov8n</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>yolov11n</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>plant detector</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>car detector</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>emotion detector</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadModelBtn_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>加载模型</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="inputLabel_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>输入选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="videoButton_5">
           <property name="toolTip">
            <string>视频video</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
	background-color: rgb(255, 255, 255);
	border:none;
}
QToolButton:hover{
	background-color: rgb(216, 216, 216);
}
QToolButton:pressed{
	background-color: rgb(165, 165, 165);
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Downloads/视频.png</normaloff>../../Downloads/视频.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pictureButton_5">
           <property name="toolTip">
            <string>图片picture</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
	background-color: rgb(255, 255, 255);
	border:none;
}
QToolButton:hover{
	background-color: rgb(216, 216, 216);
}
QToolButton:pressed{
	background-color: rgb(165, 165, 165);
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Downloads/图片.png</normaloff>../../Downloads/图片.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="cameraButton_5">
           <property name="toolTip">
            <string>摄像头camera</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
	background-color: rgb(255, 255, 255);
	border:none;
}
QToolButton:hover{
	background-color: rgb(216, 216, 216);
}
QToolButton:pressed{
	background-color: rgb(165, 165, 165);
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Downloads/摄像头.png</normaloff>../../Downloads/摄像头.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="detectBtn_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>开始检测</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="1" column="1">
          <widget class="QSlider" name="loUSlider_5">
           <property name="styleSheet">
            <string notr="true">QSlider::handle:horizontal {
	background-color: rgb(255, 170, 0);
    width: 10px;
	height:10px;	
    margin-top: -4px;  
    margin-bottom: -4px;
	border-radius:5px;
}
QSlider::sub-page:horizontal{
	background:black;
	height:3px;
}
QSlider::groove:horizontal{
	height:3px;
	background:white;
}</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="loULabel_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>loU</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="loUSpinBox_5">
           <property name="styleSheet">
            <string notr="true">QDoubleSpinBox{
	border-color: rgb(255, 255, 255);
	border-width:1px solid;
	background-color: rgb(209, 209, 209);
}</string>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0">
          <widget class="QLabel" name="confidenceLabel_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>置信度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="confSpin_5">
           <property name="styleSheet">
            <string notr="true">QDoubleSpinBox{
	border-color: rgb(255, 255, 255);
	border-width:1px solid;
	background-color: rgb(209, 209, 209);
}</string>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="confSlider_5">
           <property name="styleSheet">
            <string notr="true">QSlider::handle:horizontal {
	background-color: rgb(255, 170, 0);
    width: 10px;
	height:10px;	
    margin-top: -4px;  
    margin-bottom: -4px;
	border-radius:5px;
}
QSlider::sub-page:horizontal{
	background:black;
	height:3px;
}
QSlider::groove:horizontal{
	height:3px;
	background:white;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="0" column="0">
          <widget class="QLabel" name="delayLabel_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>帧间延时</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="delaySpinBox_5">
           <property name="styleSheet">
            <string notr="true">QDoubleSpinBox{
	border-color: rgb(255, 255, 255);
	border-width:1px solid;
	background-color: rgb(209, 209, 209);
}</string>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="delaySlider_5">
           <property name="styleSheet">
            <string notr="true">QSlider::handle:horizontal {
	background-color: rgb(255, 170, 0);
    width: 10px;
	height:10px;	
    margin-top: -4px;  
    margin-bottom: -4px;
	border-radius:5px;
}
QSlider::sub-page:horizontal{
	background:black;
	height:3px;
}
QSlider::groove:horizontal{
	height:3px;
	background:white;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="resultLabel_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>结果统计</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser_5">
           <property name="styleSheet">
            <string notr="true">QTextBrowser{
	border:1px solid;
	border-color: rgb(166, 166, 166);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(150, 150, 150);
QTabWidget{
	margin:0px;
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>实时检测</string>
       </attribute>
       <widget class="QLabel" name="videoLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>651</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>性能指标</string>
       </attribute>
       <widget class="QLabel" name="metricsLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>641</width>
          <height>541</height>
         </rect>
        </property>
        <property name="text">
         <string>                                              在此显示各项指标</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
